locale/pmod.DE.ts

```diff
@@ -889,6 +889,10 @@ export const PMOD_DE: I18nTranslation[] = [
         key: 'pmod.workflow.collapse-all',
         value: 'Alles ausblenden'
     },
+    {
+        key: 'pmod.workflow.compare',
+        value: 'Vergleiche'

```

Comment:
While the imperative is used to describe a button in english, the Germans use the infinitive. So a better translation would be "Vergleichen" or "Versionen vergleichen"

Reply:
Thanks for the suggestion to comply with naming conventions. 
With commit [c879f56](https://github.com/Xyna-Factory/xyna-process-modeller/pull/50/commits/c879f5659ac996f602cf3558e4d7b6f68142122a) now "Vergleichen" is used similar to the vertical menu bar of the modeller.